src/pika_admin.cc

```diff
@@ -115,6 +115,7 @@ void TrysyncCmd::DoInitial(PikaCmdArgsType &argv, const CmdInfo* const ptr_info)
 void TrysyncCmd::Do() {
   std::string ip_port = slash::IpPortString(slave_ip_, slave_port_);
   DLOG(INFO) << "Trysync, Slave ip_port: " << ip_port << " filenum: " << filenum_ << " pro_offset: " << pro_offset_;
+  slash::RWLock rwl(g_pika_server->rwlock(), true);
```

Comment:
This read-write lock is critical, since it will block almost all other operations. So that we expect it as small as possible.